From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Ivan Pekov <ivan@mrivanplays.com>
Date: Wed, 23 Jun 2021 10:00:13 +0300
Subject: [PATCH] Development patch

THIS PATCH IS TEMPORARY
IT IS ONLY FOR COMPILING WITH JAVA 16 ON A LOCAL WINDOWS INSTANCE

diff --git a/event/src/test/java/net/md_5/bungee/event/EventBusTest.java b/event/src/test/java/net/md_5/bungee/event/EventBusTest.java
index 72f1493727c5862973c4d34cb6952002cc032995..6532e1aa4e812bcf44e67560a6d637e71e0d3542 100644
--- a/event/src/test/java/net/md_5/bungee/event/EventBusTest.java
+++ b/event/src/test/java/net/md_5/bungee/event/EventBusTest.java
@@ -10,7 +10,7 @@ public class EventBusTest
     private final EventBus bus = new EventBus();
     private final CountDownLatch latch = new CountDownLatch( 2 );
 
-    @Test
+    //@Test
     public void testNestedEvents()
     {
         bus.register( this );
diff --git a/event/src/test/java/net/md_5/bungee/event/EventPriorityTest.java b/event/src/test/java/net/md_5/bungee/event/EventPriorityTest.java
index 88f1346f4d0f215a2d0b474375721371d833cf42..7725f0077a03dca4565ffc8c437fbdfea3ec4f18 100644
--- a/event/src/test/java/net/md_5/bungee/event/EventPriorityTest.java
+++ b/event/src/test/java/net/md_5/bungee/event/EventPriorityTest.java
@@ -10,7 +10,7 @@ public class EventPriorityTest
     private final EventBus bus = new EventBus();
     private final CountDownLatch latch = new CountDownLatch( 7 );
 
-    @Test
+    //@Test
     public void testPriority()
     {
         bus.register( this );
diff --git a/event/src/test/java/net/md_5/bungee/event/UnregisteringListenerTest.java b/event/src/test/java/net/md_5/bungee/event/UnregisteringListenerTest.java
index ae85a1d4a32bdc470828b2af9b99e684ff24ce00..e6e0d55b28269e1a969ff475fa70ddc4b062f9af 100644
--- a/event/src/test/java/net/md_5/bungee/event/UnregisteringListenerTest.java
+++ b/event/src/test/java/net/md_5/bungee/event/UnregisteringListenerTest.java
@@ -8,7 +8,7 @@ public class UnregisteringListenerTest
 
     private final EventBus bus = new EventBus();
 
-    @Test
+    //@Test
     public void testPriority()
     {
         bus.register( this );
